pkgs/development/libraries/qt-5/5.12/default.nix

```
@@ -67,6 +67,7 @@ let
         ./qtbase.patch.d/0010-qtbase-qtpluginpath.patch
         ./qtbase.patch.d/0011-qtbase-assert.patch
         ./qtbase.patch.d/0012-fix-header_module.patch
+        ./qtbase.patch.d/0014-qtbase-pkg-config.patch
```

We really need to start writing comments.

```suggestion

        # Ensure -I${includedir} in Cflags in pkg-config files.
        # See https://github.com/NixOS/nixpkgs/issues/52457
        ./qtbase.patch.d/0014-qtbase-pkg-config.patch

```

---

Agreed. PR is updated.